src/ui/UserProfile.cpp

```diff
@@ -224,6 +224,57 @@ UserProfile::refreshDevices()
     fetchDeviceList(this->userid_);
 }
 
+QVector<QString>
+UserProfile::getIgnoredUsers()
```

Comment:
This should probably be a property on the TimelineViewManager. The READ function would return the list of ignored users, while the WRITE function would modify it. the NOTIFY signal would emit a change everytime the account data event is changed.